AP-protocol.md

```diff
@@ -8,6 +8,14 @@ This document also establishes some technical requirements of the project.
 
 The **Network Initializer** reads a local **Network Initialization File** that encodes the network topology and the drone parameters and, accordingly, spawns the node threads and sets up the Rust channels for communicating between nodes.
 
+## Id Format
+
+ Each ID consists of three fields in the following format: ABN.
+
+- **A** represents the type of node: `0` for client, `1` for server, and `2` for drone.
+- **B** indicates the group that created the node (it is always `00` for clients or servers).
```

Comment:
Do we need B? And what is B for drones?

Reply:
The purpose of the field `B` is to simplify the identification of the specific group responsible for building each drone. This has several advantages:

1. **Easier Scenario Setup**: By identifying groups through field `B`, it's quicker to organize and simulate specific scenarios.

2. **Streamlined Drone Creation**: Field `B` allows for straightforward creation of new drones associated with a particular group, as they can be assigned based on the information stored in this field.

3. **Enhanced UI Display**: Having this grouping information makes it easier to display detailed information about each drone on the user interface.

Overall, `B` provides a practical way to manage and organize drones efficiently